storage/rocksdb/ha_rocksdb.cc

```diff
@@ -473,8 +473,8 @@ static MYSQL_THDVAR_BOOL(
 
 static MYSQL_THDVAR_STR(
     read_free_rpl_tables, PLUGIN_VAR_RQCMDARG | PLUGIN_VAR_MEMALLOC,
-    "List of tables that will use read-free replication on the slave "
-    "(i.e. not lookup a row during replication)",
+    "List of tables (using regex) that will use read-free replication on the "
```

Comment:
"List" suggests commas :)
I'd word this "A regexp matching the tables ... " or something similar